packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -195,11 +190,12 @@ function useProcessMedia({
       const onUploadStart = () =>
         updateExistingElements(resourceId, {
           trimData,
-          isTrimming: true,
         });
 
       const onUploadError = () =>
-        updateExistingElements(resourceId, { isTrimming: false });
+        updateExistingElements(resourceId, {
+          trimData: oldResource?.trimData || {},
```

Comment:
This should always have been this way.

Reply:
`oldResource` is not `nulll` though and we don't use optional chaining for it in other places here.

```suggestion
          trimData: oldResource.trimData || {},
```